packages/cheatsheet-local/tsconfig.json

```diff
@@ -8,13 +8,13 @@
     "allowSyntheticDefaultImports": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
-    "module": "es6",
     "target": "es5",
     "allowJs": true
   },
   "ts-node": {
     "compilerOptions": {
-      "module": "CommonJS"
+      "module": "NodeNext",
+      "moduleResolution": "NodeNext"
     }
```

Comment:
Is there a reason why there's separate ts-node compiler options here in the first place? Surely it should inherit?